application/utils.py

```
@@ -18,21 +18,27 @@
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
 
+logger = logging.getLogger(__name__)
+
+
 def int_to_bytes(num):
     return bytes([num])
 
 
 def find_free_port():
```

Ruff

Missing return type annotation for public function `find_free_port`

(ANN201)